src/app/home/home.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="wrapper">
-	<img src="{{headerImageUrl}}" alt="Loklak Cow" width="auto" height="120">
+	<img src="{{headerImageUrl}}" alt="loklak Search - Distributed Social Media Message Search Engine" title="loklak Search - Distributed Social Media Message Search Engine" width="auto" height="120">
```

Comment:
@uttu357 I know it is controlled by aspect ratio and css but from semantics and code point of view it can be done, also if in future CSS is loaded afterwards then there will be a flash of unstyled content which is not desirable. as we have noticed this lets follow the best practice here as this is a static image there is no calculation need to be done by the css it is constant and will always be constant. We can use CSS as a fallback for media queries rather than other way around. HTML is always prime and attribute styles always important.